update-bot/wlmbots/lib/commonscat_mapper.py

```diff
@@ -9,6 +10,24 @@ class CommonscatMapper(object):
 
     def __init__(self):
         self.category_cache = {}
+        self.subcategories_loaded = False
+
+
```

Comment:
There should not be double blank lines inside the class (same for other methods you're adding in this pull request).
Apparently this thing from PEP 0008 is not being checked by PyLint...